styles/styles.css

```
@@ -465,7 +465,67 @@ a.sr-focusable:focus {
   width: 1px;
 }
 
+.a11y-quicknav {
+  position: fixed;
+  margin: 0;
+  padding: 30px 0 0 20px;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
```

I think you can use the shorthand notation for this:
```suggestion
  inset: 0;
```